ui/DesignSystem/Component/Timeline.svelte

```diff
@@ -0,0 +1,17 @@
+<script>
+  import TimelineItem from "./Timeline/TimelineItem.svelte";
+  export let items = null;
+</script>
+
+<style>
+  ul {
+    margin-top: 24px;
+  }
+</style>
+
+<ul>
+  {#each items as item}
+    <TimelineItem {item} />
+  {/each}
+  <TimelineItem />
```

Comment:
This looks like a leftover.

Reply:
It wasn't but I understand why it looked like this. I made it all a bit more explicit now.